robottelo/cli/import_.py

```diff
@@ -350,11 +355,13 @@ def repository_with_tr_data(cls, options=None):
         transition_data = []
         if result.return_code == 0:
             transition_data = [
-                cls.read_transition_csv(ssh.command(command).stdout[:-1])
-                for command
+                cls.read_transition_csv(
+                    ssh.command(cmd[0]).stdout[:-1], cmd[1]
+                )
+                for cmd
```

Comment:
You can also write something like:

```python
cls.read_transition_csv(ssh.command(foo).stdout[:-1], bar)
for foo, bar
in (
    (two_element, iterable),
    (two_element, iterable),
)
```

I think that would be more clear than `cmd[0]` and `cmd[1]`.